### ISSUE TITLE
Media component

### ISSUE THREAD

**Issue by Kyzyl-ool** [gravity-ui/page-constructor#486]
One proposes the following changes to image's alt attribute as part of accessibility improvement:

1. Currently all alt attributes are set as "Image alt". By default, they should be empty.
2. Each component / block, using the alt attribute, should have an opportunity to set custom alt value from props
1. Add `aria-label="Close"` for close button in FullscrrenImage
